selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/projectexplorer/dependencies/TransitiveDependencyTest.java

```diff
@@ -92,10 +99,17 @@ public void transitiveDependencyTest() throws Exception {
 
   private void deleteDependency() {
     editor.waitActive();
-    loader.waitOnClosed();
-    for (int i = 36; i <= 40; i++) {
-      editor.setCursorToLine(i);
-      editor.selectLineAndDelete();
-    }
+    Actions action = actionsFactory.createAction(seleniumWebDriver);
```

Comment:
What about moving `actionsFactory.createAction(seleniumWebDriver)` to the **SeleniumWebDriverHelper** class?